### PR TITLE
APP-14422: Create Accordion component

### PR DESCRIPTION
**Pull request by orelHAnyvision**
### Checklist
- [ ] Does this need a new/update storybook example?
- [ ] Does this need new/update tests?

- [ ] run eslint
- [ ] run test
- [ ] run build

If your answer is yes to any of these, please make sure to include it in your PR.

> NOTE: Please submit all PRs to the `development` branch.

### CODE REVIEW

File: src/components/Accordion/Accordion.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import classNames from 'classnames'
+import { ArrowRight } from '@anyvision/anv-icons'
+import { Tooltip } from '../Tooltip'
+import styles from './Accordion.module.scss'
+
+const Accordion = ({
+  className,
+  data = [],
+  expandAll = false,
+  disableAll = false,
```
Comment (AdimAnyvision) [inline src/components/Accordion/Accordion.js:12]: you already added default values in defaultProps, so remove

File: src/components/Accordion/Accordion.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+  const handleItemSelected = idx => {
+    if (disableAll) {
+      return
+    }
+    if (selected[idx]) {
+      setSelected(prev => ({ ...prev, [idx]: false }))
+    } else {
+      setSelected(prev => ({ ...prev, [idx]: true }))
+    }
```
Comment (AdimAnyvision) [inline src/components/Accordion/Accordion.js:35]:
```suggestion
   setSelected(prev => ({ ...prev, [idx]: ![selected[idx] }))
```

File: src/components/Accordion/Accordion.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  }
+
+  return (
+    <div className={classNames(styles.accordion, className)}>
+      {data.map(({ title, content, disabled }, idx) => {
+        const isActive = selected[idx]
+        return (
+          <div
+            key={idx}
```
Comment (AdimAnyvision) [inline src/components/Accordion/Accordion.js:44]: Please change key to title, so it will be unique.. maybe `${idx}-${title}` if title is not unique

Comment (orelHAnyvision) [reply src/components/Accordion/Accordion.js:44]: @AdimAnyvision title could be duplicated, and could be element... I can add id property to the item object

**Review by AdimAnyvision — APPROVED** [review]

**Pull request merged**